pmedianetwork/eslint-config
# Feature/41-update-eslint

close PR #41 

Update 
- `eslint` 
- `eslint-config-prettier`
- `@typescript-eslint/eslint-plugin`
- `@typescript-elint/parser`
- `babel-eslint`

**How to test**

- Create a dependency on this branch in the following repositories and do `npm run lint` (or `npm run eslint`, depending on the repo)
  - insights
  - datatap
  - presense
  - adverity-components

--> There should be no eslint runtime errors (e.g. due to misconfiguration)

If there are no additional eslint errors popping up, this can be a `patch` version. Otherwise, we should make a major version.
